routes/global_challenge.py

```diff
@@ -53,14 +62,20 @@ def get_individual_baseline(meter_id):
     """
 
     # pylint: disable=line-too-long
-    query = "SELECT timestamp, baseline FROM base_line WHERE meter_id = '%s' ORDER BY timestamp DESC" % meter_id
+    # query = "SELECT timestamp, baseline FROM base_line WHERE meter_id = '%s' ORDER BY timestamp DESC" % meter_id
 
     try:
         engine = get_engine()
-        with engine.connect() as con:
+        with engine.connect():
```

Comment:
Siehe oben. Du musst auch unbedingt checken, ob Deine Funktionen dann auch das Richtige tun, d.h. die richtigen Werte in die Datenbank schreiben. Am besten suchst Du Dir dazu die entsprechenden Stellen in den Unit Tests und läst Dir mal ein paar Werte ausgeben. Oder schreibst ein paar neue Tests, wenn die Funktionen noch nicht abgedeckt sein sollten. Wenn Du dabei Hilfe brauchst, komm einfach auf mich zu. :) 
Die Datenbank, die in den Unit Tests verwendet wird, ist zwar in-memory, also flüchtig, ist aber genauso strukturiert wie die echte.